lib/src/code/code.dart

```diff
@@ -177,19 +209,96 @@ class Code {
     }
   }
 
+  ///Assumes that there are only one visible section.
+  static Map<String, HiddenRange> _visibleSectionsToHiddenRanges(
+    Set<String> names,
+    Map<String, NamedSection> namedSections,
+    CodeLines lines,
+  ) {
+    final section = namedSections
+        .toVisibleSections(
+          lines: lines.lines,
+          visibleSectionsNames: names,
+        )
+        .firstOrNull;
+
+    final result = <String, HiddenRange>{};
+
+    if (section == null) {
+      return result;
+    }
+
+    late int beginnigEnd;
+    late int beginnigLastLine;
+    final startLine = lines.lines[section.startLine];
+    if (startLine.text.substring(startLine.indent).startsWith('// [START')) {
```

Comment:
Remove the two branches. The visible section always starts on the line where the section starts.

Reply:
+